test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -1234,7 +1234,7 @@ void ProtoConverter::visit(CaseStmt const& _x)
 
 void ProtoConverter::visit(SwitchStmt const& _x)
 {
-	if (_x.case_stmt_size() > 0 || _x.has_default_block())
```

Comment:
The original motivation for this condition was the following logic:

> If a switch statement has at least one case OR a default block, process it.

We can break this down into the following permutations for a switch statement
  1. at least one case + default
  2. no case + default
  3. at least one case + no default
  4. no case + no default

I believe this PR only throws a warning for `2` (i.e., no switch case present but default present). For this specific permutation, the fuzzer (without the proposed modification) would generate a test case such as the following:

```
let x
switch x
  default {}
```

Which is okay for the `develop` branch for the same reason that retaining `pc()` (please see ) is okay. However, as we did in the `pc()` instance, we could apply this patch to `breaking` branch when we error on switch cases with only default. At that time, this logic makes complete sense because we should not generate a switch case with only a default case. Or, in other words, we should only generate a default case in conjunction with at least one case statement.

tl;dr: Please revert this change as fuzzing switch statements with only a default block is still relevant for the develop branch. Please consider applying this patch in a futute PR that will error on switch statements with only a default case :-)

Reply:
@bshastry We don't plan to introduce errors for switch with only a default. But if warnings are fine, then I guess, this change can be reverted.